scripts/fn_paradrop.sqf

```diff
@@ -0,0 +1,213 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Josh Vee, Jonpas, Veteran29
+ * An all-purpose paradrop function
+ * Call from vehicle waypoint with (isServer) check.
+ * 
+ * The vehicle will set move waypoints on two points of a circle surrounding 
+ * a marker and initiate the drop at the fist waypoint.  Can be called multiple
+ * times for different groups loaded on the same vehicle, but optimally
+ * used either for single group on helicopter or multiple groups on planes.
+ *
+ * 
+ * 
+ * It inserts waypoints in front of the one in which it is called
+ * therefore it can be used with multiple waypoints without issue.
+ *
+ * Known behavior: 
+ * 1. Locked planes/helicopters result in no drops.
+ * 2. Planes/helicopters in combatMode "COMBAT" have altitude overrides.
+ * 3. Drop distribution ultimately depends on AI pathing, and AI is stupid.
+ * 4. Function is designed to be called synchronously with craft reaching the
+ * WP from which it is called; if it is held up, then the drop WPs will not
+ * reflect the current azimuth.  If synchronizing WPs, recommend you sync WPs
+ * BEFORE the one that calls this function to avoid weird behavior, especially
+ * if the drop marker is close by.
+ *
+ * Optional Arguments: Drop Mode
```

Comment:
Instead of using a number for this, it could also be a string, eg. "modern", "danger", "halo", "civ", etc. Just a different representation of it. Either is fine, both need good documentation.

Reply:
I understand; my intention was to use whatever convention we had in place so I defaulted to using the one from the artillery script where ammunition type is denoted by number.